codechecker_common/tests/unit/test_gerrit_converter.py

```diff
@@ -21,7 +21,21 @@
 class TestReportToGerrit(unittest.TestCase):
     @classmethod
     def setup_class(cls):
-        cls.severity_map = {"my_checker": "LOW"}
+        class CheckerLabels:
+            def severity(self, checker):
+                if checker == 'my_checker':
+                    return 'LOW'
+                else:
+                    # This assertion warns when a new test-case in the future
+                    # intends to query the severity of another checker. The
+                    # original behavior of this function is to return the
+                    # defult 'UNSPECIFIED' value by defult when the severity is
+                    # not provided in the config file.
+                    assert False, \
+                        'Currently no test-case quieries other labels for ' \
+                        'other checkers.'
```

Comment:
We can save an indentation here:
```suggestion
                if checker == 'my_checker':
                    return 'LOW'

                # This assertion warns when a new test-case in the future
                # intends to query the severity of another checker. The
                # original behavior of this function is to return the
                # defult 'UNSPECIFIED' value by defult when the severity is
                # not provided in the config file.
                assert False, \
                    'Currently no test-case quieries other labels for ' \
                    'other checkers.'
```